pontoon/base/tests/managers/test_entity.py

```diff
@@ -650,6 +650,107 @@ def test_mgr_entity_filter_rejected_plural(resource_a, locale_a):
     }
 
 
+@pytest.mark.django_db
+def test_mgr_entity_filter_missing_without_unreviewed(resource_a, locale_a):
+    entities = [
+        EntityFactory.create(resource=resource_a, string="testentity%s" % i,)
+        for i in range(0, 5)
+    ]
+
+    TranslationFactory.create(
+        locale=locale_a, entity=entities[0], approved=False, fuzzy=False, rejected=True,
+    )
+    TranslationFactory.create(
+        locale=locale_a, entity=entities[0], approved=False, fuzzy=False, rejected=True,
+    )
+    TranslationFactory.create(
+        locale=locale_a, entity=entities[2], approved=True, fuzzy=False, rejected=False,
+    )
+    TranslationFactory.create(
+        locale=locale_a, entity=entities[3], approved=False, fuzzy=True, rejected=False,
+    )
+    TranslationFactory.create(
+        locale=locale_a, entity=entities[1], approved=False, fuzzy=False, rejected=True,
+    )
+    TranslationFactory.create(
+        locale=locale_a,
+        entity=entities[1],
+        approved=False,
+        fuzzy=False,
+        rejected=False,
+    )
```

Comment:
Hmmm, I find it weird that black formats this block differently than the previous ones. 👀

Reply:
Length of string probably

Reply:
Ah, now I see it's one more character in this line. Good point!